.github/workflows/test.yml

```
@@ -1,29 +1,141 @@
+  #         static_site_generator: next
+
+  #     - name: Upload Pages Artifact
+  #       if: always()
+  #       uses: actions/upload-pages-artifact@v3
+  #       with:
+  #         path: playwright-report/
+
+  # deploy:
+  #   needs: e2e-tests
+  #   runs-on: ubuntu-latest
+    
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)